### PR TITLE
LPS-198121 [Marketplace] [Impedibug] Adjustments on the Next Steps page

### PR DESCRIPTION
**Pull request by claricebrasil**

### CODE REVIEW

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/NextStepPage/NextStepPage.tsx [hunk 1]
```diff
@@ -114,7 +110,8 @@ export function NextStepPage({
 		[TypeLicense.PAID]: (
 			<Header
 				description={
-					<>
+				
+						<>
 						<p>
```
Comment (silasbsi) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/NextStepPage/NextStepPage.tsx:115]: could you format this piece of code, please?
```suggestion
				
					<>
						<p>
```

**Review by silasbsi — CHANGES_REQUESTED** [review]
I added one suggestion

**Review by joanagrego — APPROVED** [review]

**Review by silasbsi — APPROVED** [review]
Good job @claricebrasil Thanks :martial_arts_uniform:

**Comment by silasbsi** [comment]
ci:forward:force

**Comment by claricebrasil** [comment]
ci:forward:force

**Comment by silasbsi** [comment]
@LuanaKSilva can you see if the errors are related with what was developed in this PR?

**Comment by LuanaKSilva** [comment]
Hey @claricebrasil and @silasbsi, I analyzed the flaws, and none of them have anything to do with PR.

**Comment by claricebrasil** [comment]
ci:forward:force

**Comment by silasbsi** [comment]
Just starting reviewing :)

**Comment by silasbsi** [comment]
Resend: https://github.com/brianchandotcom/liferay-portal/pull/141877

**Pull request closed without merging**